openqaoa/optimizers/result.py

```diff
@@ -62,30 +62,30 @@ def __init__(
         self.cost_hamiltonian = cost_hamiltonian
 
         self.evals = {
-            "number of evals": log.func_evals.best[0],
-            "jac evals": log.jac_func_evals.best[0],
-            "qfim evals": log.qfim_func_evals.best[0],
+            "number_of_evals": log.func_evals.best[0],
+            "jac_evals": log.jac_func_evals.best[0],
+            "qfim_evals": log.qfim_func_evals.best[0],
         }
 
         self.intermediate = {
-            'angles log': np.array(log.param_log.history).tolist(),
-            'intermediate cost': log.cost.history,
-            'intermediate measurement outcomes':
+            'angles_log': np.array(log.param_log.history).tolist(),
+            'intermediate_cost': log.cost.history,
+            'intermediate_measurement_outcomes':
                 log.measurement_outcomes.history,
-            'intermediate runs job id': log.job_ids.history
+            'intermediate_runs_job_id': log.job_ids.history
         }
 
         self.optimized = {
-            'optimized angles':
+            'optimized_angles':
                 np.array(log.param_log.best[0]).tolist()
                 if log.param_log.best != [] else [],
-            'optimized cost':
+            'optimized_cost':
                 log.cost.best[0]
                 if log.cost.best != [] else None,
-            'optimized measurement outcomes':
+            'optimized_measurement_outcomes':
```

Comment:
As a general remark: I still think that having an `optimized_measurement_outcomes` and getting a wave-function is quite misleading. But I think we can probably still live with it ...

Reply:
I also think it is a little misleading, but how could we change that? I don't think it is a good idea to change the key depending on the type of backend

Reply:
No, it isn't. Let's leave it like this for the moment